src/Stratis.Bitcoin.Features.Wallet/WalletSyncManager.cs

```diff
@@ -214,6 +236,44 @@ public virtual void ProcessBlock(Block block)
             this.logger.LogTrace("(-)");
         }
 
+        /// <inheritdoc />
+        public virtual void ProcessBlock(Block block)
+        {
+            Guard.NotNull(block, nameof(block));
+
+            this.logger.LogTrace("({0}:'{1}')", nameof(block), block.GetHash());
+
+            if (!this.walletManager.ContainsWallets)
+            {
+                this.logger.LogTrace("(-)[NO_WALLET]");
+                return;
+            }
+
+            if (block.BlockSize != null)
+                this.blocksQueueSize += block.BlockSize.Value;
```

Comment:
You must move this to line 272, you only want to increase when you actually enqueue after that. So down below if you want to enqueue, first increase the size, then enqueue. But we cannot increase the queue size for blocks that we don't enqueue

Reply:
Ar yes, my mistake - will move down.